library/kotlin/src/io/envoyproxy/envoymobile/ResponseTrailersBuilder.kt

```diff
@@ -0,0 +1,28 @@
+package io.envoyproxy.envoymobile
+
+/*
+ * Builder used for constructing instances of `ResponseTrailers`.
+ */
+class ResponseTrailersBuilder: HeadersBuilder {
+  /**
+   * Initialize a new instance of the builder.
+   */
+  constructor() : super(mutableMapOf())
+
+  /**
+   * Instantiate a new builder. Used only by ResponseTrailers to convert back to
+   * ResponseTrailersBuilder.
```

Comment:
Do we need to mention this implementation detail since this is internal anyway?

Reply:
We have this call out in the `RequestTrailersBuilder`